src/main/java/com/teamc/fms/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.teamc.fms.controller;
+
+import com.teamc.fms.dto.UserDto;
+import com.teamc.fms.entity.User;
+import com.teamc.fms.mapper.UserMapper;
+import com.teamc.fms.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/v1/admin")
+@RequiredArgsConstructor
+public class UserController {
```

Comment:
We use UserController as an abstract over admin and tenant, when for example we need to blockUser, so this API will be applied in both admin and tenant
But here we create admin, so this API should be in AdminController